docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: "3.8"
+
+services:
+  db:
+    image: mysql:8
+    environment:
+      MYSQL_ROOT_PASSWORD: gtsb-wp-dckr
+      MYSQL_DATABASE: wordpress
+      MYSQL_USER: wordpress
+      MYSQL_PASSWORD: gtsb-wp-dckr-user
+    restart: always
+    volumes:
+      - "db_data:/var/lib/mysql"
+      - "./docker/seed:/docker-entrypoint-initdb.d"
+
+  wordpress:
+    depends_on:
+      - db
+    # image: duffn/wordpress-wpgraphql-wpgatsby:latest
+    # While pulling the image will be faster, if you like, you can build the
+    # image locally, but uncommenting the lines below and commenting out the
+    # `image` line above.
+    # The image above will be kept up-to-date, however, you can change WPGraphQL
+    # and WPGatsby versions below if you build yourself.
+    build:
+      context: ./docker/wordpress
+      args:
+        WPGRAPHQL_VERSION: v0.13.3
```

Comment:
this is where we have to pin the prebuild docker image plugin versions (more ideal for speed)

build args don't accept `env_file` yet unfortunately, but maybe there's a way around that issue i'm overlooking? @jasonbahl i saw you using template replacement in the wp-graphql docker compose, how is that compiled? is that just a new docker-compose feature i'm unaware of?

however, the wp-cli image runs everything when the container runs, so we can use an env file to specify those plugin versions